### DISCUSSION TITLE
Application recording?

### DISCUSSION THREAD

**Discussion by johansenha** [gravitational/teleport#8838] [Q&A]
Hello, Is there a application recording (Web UI recording) available in the current release either community or enterprise (or both?). We have tested it with the community edition but couldn't find the possibility to record an web application call. That's one of the major feature we would need to start with.

Thanks in advance. 
Jo

**Comment by webvictim** [comment]
This functionality is not currently present in either OSS or Enterprise versions of Teleport. I believe there is a feature request for this somewhere, but the work is not currently scheduled.
